internal/conduit/pipeline/oracle.go

```diff
@@ -0,0 +1,95 @@
+package pipeline
+
+import (
+	"context"
+	"log"
+
+	"github.com/base-org/pessimism/internal/conduit/models"
+)
+
+// OracleType ...
+type OracleType = string
```

Comment:
probably better in a "types.go" kinda file.

Reply:
[1e84ea9](https://github.com/base-org/pessimism/pull/3/commits/1e84ea94c6a1408b28cb88974fa6ab6332315ce5)